Gifski/MainWindowController.swift

```diff
@@ -258,6 +258,7 @@ final class MainWindowController: NSWindowController {
 					self.presentError(error, modalFor: self.window)
 				}
 
+				try? outputUrl.setMetadata(key: .itemCreator, value: "\(App.name) \(App.versionWithBuild)")
```

Comment:
Been thinking about it. I don't think we should include the build number. It's not useful for users.
```suggestion
				try? outputUrl.setMetadata(key: .itemCreator, value: "\(App.name) \(App.version)")
```